core/templates/dev/head/pages/story-editor-page/editor-tab/story-node-editor.directive.html

```diff
@@ -26,8 +26,11 @@
   </div>
   <label style="padding-top: 2em;" class="form-heading"> Exploration ID </label>
   <form class="form-horizontal" role="form" ng-submit="updateExplorationId(explorationId)">
-    <input class="protractor-test-exploration-id-input" type="text" ng-model="explorationId" ng-change="checkCanSaveExpId()">
+    <input class="protractor-test-exploration-id-input" type="text" ng-model="explorationId" ng-change="checkCanSaveExpId(); showInvalidExpError = false;">
```

Comment:
We usually don't initialize variables in HTML files. Can you do this in the ng-change function itself?

Reply:
Done!